static/app/components/sidebar/index.tsx

```diff
@@ -180,7 +191,13 @@ function Sidebar({location, organization}: Props) {
       <SidebarItem
         {...sidebarItemProps}
         icon={<IconLightning size="md" />}
-        label={<GuideAnchor target="performance">{t('Performance')}</GuideAnchor>}
+        label={
+          <GuideAnchor target="performance">
+            {organization?.experiments?.APMSidebarExperiment
```

Comment:
You can use `experimentAssignment` which is returned from `useExperiment`.

Also there is the possible return value of `-1`, so I think you'd want to do

```ts
experimentAssignment === 1 ? t('APM') : t('Performance')
```